DataFormats/Common/interface/ValueMap.h

```diff
@@ -14,6 +14,7 @@
 #include <map>
 #include <iterator>
 #include <algorithm>
+#include <stddef.h>
```

Comment:
@Dr15Jones - just curious why later cstddef is included and not here:
https://github.com/cms-sw/cmssw/pull/19773/files#diff-1588000c7ee3060dfd82d6c7ea0c1b4bR8